app.js

```diff
@@ -16,10 +16,10 @@ app.set('views', path.join(__dirname, 'views'));
 app.set('view engine', 'jade');
 
 // uncomment after placing your favicon in /public
-//app.use(favicon(path.join(__dirname, 'public', 'favicon.ico')));
+// app.use(favicon(path.join(__dirname, 'public', 'favicon.ico')));
```

Comment:
Lines 18 and 19 can be deleted as most likely they won't be used and if they are needed, they can be recovered from history